source/openstack/guests.rst

```diff
@@ -100,3 +98,7 @@ AARCH64:
       $ source openrc
       $ glance image-create --name $DISTRO --disk-format=qcow2 \
         --container-format=bare < $DISTRO-compressed.qcow2
+
+.. rubric:: Footnotes
+
+.. [#ubuntu] We are currently fixing some issues with our Ubuntu 20.04 images.
```

Comment:
I just created this issue https://github.com/osuosl/packer-templates/issues/124 which might be good to link here so folks can track it's progress.